plugins/event_source/cmd.py

```diff
@@ -0,0 +1,102 @@
+"""
+cmd.py
+
+An ansible-rulebook event source plugin for running a custom command.
+It can send events to the rule engine from the stdout.
+
+Arguments:
+    command: str
+        The string with the command to execute
+    send_output: bool, default: false
+        Send events from the stdout.
+        Each event must be a serialized JSON string
+        ended with a newline character ('\n')
+
+Examples:
+---
+- name: Example with output proccessing
+  hosts: all
+  sources:
+    - ansible.eda.cmd:
+        command: node my_custom_plugin.js
+        send_output: true
+  rules:
+    - name: r1
+      condition: event.cmd.status == "Running"
+      action:
+        print_event:
+
+
+---
+- name: Example combined with webhook plugin
+  hosts: all
+  sources:
+    - ansible.eda.webhook:
+        port: 6080
+    - ansible.eda.cmd:
+        command: node my_custom_plugin.js --url http://localhost:6080/eda
+  rules:
+    - name: r1
+      condition: event.payload.status == "Running"
+      action:
+        print_event:
+"""
+
+import asyncio
+import json
+import logging
+import shlex
+from typing import Any, Dict
+
+LOGGER = logging.getLogger("ansible.eda.cmd")
+
+
+async def process_output(
+    proc: asyncio.subprocess.Process, queue: asyncio.Queue
+):
+    # type hint warn: proc.stdout can be None, this is not our case
+    while output := await proc.stdout.readline():
+        try:
+            event = json.loads(output.decode())
+            await queue.put({"cmd": event})
+        except json.JSONDecodeError as e:
+            LOGGER.error("Can not deserialize JSON data: %s", e)
+
+
+async def main(queue: asyncio.Queue, args: Dict[str, Any]):
+    command = str(args["command"])
+    send_output = False
+    stdout_mode = asyncio.subprocess.DEVNULL
+
+    if "send_output" in args:
+        send_output = args["send_output"]
+
+    if send_output:
+        stdout_mode = asyncio.subprocess.PIPE
+
+    cmd_tokens = shlex.split(command)
+    proc = await asyncio.subprocess.create_subprocess_exec(
+        cmd_tokens[0],
+        *cmd_tokens[1:],
+        stdout=stdout_mode,
+        stderr=asyncio.subprocess.PIPE,
+    )
+
+    LOGGER.info("Process started")
+
+    if send_output:
+        _unused, stderr = await asyncio.gather(
```

Comment:
Minor: It looks that project's pylint rules can be adjusted. It's completely acceptable to use a single underscore (`_` ) for an unused variables.

Reply:
Hi @cutwater This is not adjustable. It not comes from pylint directly but by ansible-test which is implemented in the pipelines following the [ansible guidelines](https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_testing.html)

It could be ignored adding the proper comment in the code and I thought like you, but I opened a ticket in the community and their reasons have convinced to me: 
https://github.com/ansible/ansible/issues/79646

Reply:
Ok, makes sense, but according to the explanation provided, it's better to give it a meaningful name. Like `_stdout`.

Reply:
agreed. Updated